docs/md-vs-ipynb.md

```diff
@@ -0,0 +1,31 @@
+# How to choose between `.ipynb` and `.md` when writing Jupyter notebooks
+
+You can work with Jupyter notebooks in both the `.ipynb` format as well as in a text-based format like [MyST Markdown](./notebooks-with-markdown.md). This is a brief guide to help you choose when to use one vs. the other.
```

Comment:
Maybe add one more sentence that working in jupyper/jupyterlab the user experience will be the same, they don't usually notice the difference between the formats assuming the required dependencies, jupytext and maybe jupyerlab-myst, are installed.